Formula/h/handbrake.rb

```diff
@@ -56,6 +54,30 @@ def install
 
     inreplace "contrib/ffmpeg/module.defs", "$(FFMPEG.GCC.gcc)", "cc"
 
+    if OS.linux? && Hardware::CPU.arm?
+      # Disable SVE2 for ARM builds, as it causes issues with the x265 module.
+      inreplace ["contrib/x265_10bit/module.defs", "contrib/x265_12bit/module.defs", "contrib/x265_8bit/module.defs"],
+                "-DENABLE_CLI=OFF",
+                "-DENABLE_CLI=OFF -DENABLE_SVE2=OFF"
+
+      # Fix AArch64 assembly for pixel-util.S
+      (buildpath/"contrib/x265/A09-aarch64-fix.patch").write <<~PATCH
+        diff --git a/source/common/aarch64/pixel-util.S b/source/common/aarch64/pixel-util.S
+        index e2b31e4..1bcaf4a 100644
+        --- a/source/common/aarch64/pixel-util.S
+        +++ b/source/common/aarch64/pixel-util.S
+        @@ -860,7 +860,7 @@ function PFX(scanPosLast_neon)
+             lsl             w13, w13, w6
+             lsl             w15, w15, w6
+             extr            w14, w14, w13, #31
+        -    bfc             w15, #31, #1
+        +    bfm             w15, wzr, #31, #31
+             cbnz            w15, .Loop_spl_1
+         .Lpext_end:
+             strh            w14, [x2], #2
+      PATCH
+    end
```

Comment:
It would be simpler if we could just get this to use Homebrew `x265`, so we don't have to fix this issue in multiple places. Is there no way to do that here?

Reply:
I could try, but they publish their own version for many dependencies and this make it more harder than think.